### PR TITLE
Set the host app version to the context tags. #183

### PR DESCRIPTION
**Pull request by gabsanchez**
#183 Sets the host app version to the context tags, so we don't rely on the package.json.
Team:
@gabsanchez
@ronaldour
@childmario95
@tohka53
@mariopg

Engineers in charge:
@oscargarciacolon
@gzepeda

### CODE REVIEW

File: Library/Client.ts [hunk 1]
```diff
@@ -376,6 +377,14 @@ class Client {
 
         return newTags;
     }
+    /**
+     * Sets the client app version to the contest tags.
```
Comment (OsvaldoRosado) [inline Library/Client.ts]: Nit: typo on contest > context

File: Library/Client.ts [hunk 2]
```diff
@@ -376,6 +377,14 @@ class Client {
...
+     * @param version, takes the host app version.
+     */
+    public setVersion(version: string)
```
Comment (OsvaldoRosado) [inline Library/Client.ts]: What do you think of `overrideApplicationVersion` ?

**Comment by OsvaldoRosado** [comment]
Thanks for submitting this!

**Comment by gabsanchez** [comment]
@OsvaldoRosado thank you for your feedback, please let me know if there's anything else I can do to complete this pull request.

File: Tests/Library/Client.tests.ts [hunk 3]
```diff
@@ -805,4 +805,10 @@ describe("Library/Client", () => {
             assert.ok(!processorExecuted, "telemetry processor should NOT be executed");
         });
     });
+    describe("#setVersion()", () => {
```
Comment (OsvaldoRosado) [inline Tests/Library/Client.tests.ts]: The method name should be updated here as well (string and the method call below)

**Comment by gzepeda** [comment]
Kudos to the team!

**Pull request merged**